src/common/types.cpp

```diff
@@ -1,14 +1,30 @@
-// local includes
+// header include
```

Comment:
`local` just means non system and non dependency includes. These lines probably aren't necessary if you want to remove them. It's just something I tend to do, especially in Python projects.

Reply:
We follow the same pattern at work, but we also explicitly separate the main header include, so personally like them